client/src/js/samples/components/Create/__tests__/ReadSelector.test.js

```diff
@@ -0,0 +1,80 @@
+import ReadSelector from "../ReadSelector";
+
+describe("<ReadSelector />", () => {
+    let props;
+    let state;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['state' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4488032)